### DISCUSSION TITLE
Audit Logs not displayed in the UI

### DISCUSSION THREAD

**Discussion by ar3diu** [reflexsoar/reflex-api#195] [Q&A] [answered]
Has anyone encountered this problem?

Version: 2022.05.01
Backend: Elasticsearch
Browser: Brave, Firefox

![image](https://user-images.githubusercontent.com/61625853/174312686-63ad1d91-8584-4de2-989b-629349c466ad.png)

**Comment by n3tsurge** [comment, marked as answer]
This is a Bug in our Approved IP feature which restricts what IP addresses can use the platform.  I found the issue in code, will release a bug fix for it.

**Reply by ar3diu** [reply]
Thank you very much for your answer, Brian! :raised_hands:
